app/components/schema-field-select/component.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  init() {
+    this._super(...arguments);
+
+    let key = this.get('key');
+    let document = this.get('document');
+    let defaultValue = this.get('property.default');
+    let initialValue = document.get(key) || defaultValue || '';
```

Comment:
This chunk of code seems to be used in all the components, can we extract to a util function that they can all share?

```
export default function(component) {
   let key = component.get('key');
   let document = component.get('document');
   let defaultValue = component.get('property.default');
   let initialValue = document.get(key) || defaultValue || '';

   component.set('value', initialValue);
}
```